apps/central/src/components/Groups.vue

```diff
@@ -127,7 +141,28 @@ export default {
         );
       }
       return filtered.sort((a, b) => a.name.localeCompare(b.name));
-    }
+    },
+    showChangeColumn() {
+      return (
+        this.session.email == "admin" ||
+        (this.session &&
+          this.session.roles &&
+          this.session.roles.includes("Manager"))
+      ) && this.changelogSchemas;
+    },
+    changelogSchemas() {
+      if (
+        this.session &&
+        this.session.settings &&
+        this.session.settings["CHANGELOG_SCHEMAS"]
+      ) {
+        return this.session.settings["CHANGELOG_SCHEMAS"]
+          .split(",")
```

Comment:
as a future todo we must think how we can move this setting somehow to schema creation process. Now I believe one must first create the setting and then create the schema correct?

Reply:
yes if the keep this impl a agree it needs to added to the api ( and ui ) zo that the change log feature cna be turned on ( create change logtable , en setup trigger) and off ( turn of trigger and maybe drop changelog table )